packages/signature/src/lib/__test__/unit/util.spec.ts

```diff
@@ -58,6 +65,15 @@ const k1Jwk: Jwk = {
   d: 'ENTv9xjMG6RkHwamtOk3l0mlHOZN7herKFEmGPPKK04'
 }
 
+const eddsaKey: Jwk = {
```

Comment:
```suggestion
const eddsaPrivateKey: Jwk = {
```

Make it clear it's a private key, please.

Reply:
It's both, it holds private and public material (it has 'd' AND 'x')